src/CMakeLists.txt

```
@@ -78,6 +78,9 @@ set(SOURCE_FILES
         model/DataModel.hxx
         model/ButtonModel.{hxx,cxx}
         model/MouseState.{hxx,cxx}
+		model/iLocalization.hxx
+		model/Localization.{hxx,cxx}
+		model/SettingsModel.{hxx,cxx}
```

Bad indentation. Please diable "TAB" from your editor
```suggestion
        model/iLocalization.hxx
        model/Localization.{hxx,cxx}
        model/SettingsModel.{hxx,cxx}
```